.conda/README.md

```diff
@@ -0,0 +1,32 @@
+# Publish to conda
+
+There is two way of publishing to conda:
+- A fully automatic in CI that publish to an "openfisca" channel. See below for more information.
+- A more complex for Conda-Forge.
+
+We use both for openfisca-core but only _openfisca channel_ for _country-template_.
+
+## Automatic upload
+
+The CI automaticaly upload the PyPi package, see the `.github/workflow.yml`, step `publish-to-conda`.
+
+## Manual actions made to make it works the first time
+
+- Create an account on https://anaconda.org.
+- Create a token on https://anaconda.org/openfisca/settings/access with _Allow write access to the API site_. Warning, it expire on 2023/01/13.
+- Put the token in a CI env variable ANACONDA_TOKEN.
+
+## Manual actions before CI exist
+
+To create the package you can do the following in the project root folder:
+
+- Edit `.conda/meta.yaml` and update it if needed:
+    - Version number
+    - Hash SHA256
+    - Package URL on PyPi
+
+- Build & Upload package:
+    - `conda install -c anaconda conda-build anaconda-client`

```

Comment:
This command brings `python 3.9` and this seems to come from `meta.yaml`, `requirements` value. 
From previous tests, it shouldn't be a technical issue but for now, the country-template goes from python 3.6 to 3.8 (from `setup.py`) and the `publish-to-conda` target in CI runs with python 3.7.9. Make the same choice of python revision on all steps?

Extract from `conda install` command:  

```shell
The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    anaconda-client-1.9.0      |   py39hecd8cb5_0         150 KB  anaconda
    attrs-21.4.0               |     pyhd3eb1b0_0          49 KB  anaconda
    beautifulsoup4-4.11.1      |   py39hecd8cb5_0         183 KB  anaconda
    brotlipy-0.7.0             |py39h9ed2024_1003         359 KB  anaconda
    bzip2-1.0.8                |       h1de35cc_0         104 KB  anaconda
    ca-certificates-2022.4.26  |       hecd8cb5_0         132 KB  anaconda
    certifi-2021.10.8          |   py39hecd8cb5_2         156 KB  anaconda
    cffi-1.15.0                |   py39hc55c11b_1         220 KB  anaconda
    chardet-4.0.0              |py39hecd8cb5_1003         205 KB  anaconda
    charset-normalizer-2.0.4   |     pyhd3eb1b0_0          33 KB  anaconda
    clyent-1.2.2               |   py39hecd8cb5_1          20 KB  anaconda
    conda-4.12.0               |   py39hecd8cb5_0        16.9 MB  anaconda
    conda-build-3.21.8         |   py39hecd8cb5_2         577 KB  anaconda
    conda-package-handling-1.8.1|   py39hca72f7f_0         1.6 MB  anaconda
    cryptography-36.0.0        |   py39hf6deb26_0         1.3 MB  anaconda
    filelock-3.6.0             |     pyhd3eb1b0_0          12 KB  anaconda
    glob2-0.7                  |     pyhd3eb1b0_0          12 KB  anaconda
    icu-58.2                   |       h0a44026_3        22.6 MB  anaconda
    idna-3.3                   |     pyhd3eb1b0_0          55 KB  anaconda
    jinja2-3.0.3               |     pyhd3eb1b0_0          99 KB  anaconda
    jsonschema-4.4.0           |   py39hecd8cb5_0         120 KB  anaconda
    jupyter_core-4.9.2         |   py39hecd8cb5_0          84 KB  anaconda
    libarchive-3.4.2           |       ha0e9c3a_0         1.6 MB  anaconda
    libcxx-12.0.0              |       h2f01273_0         1.1 MB  anaconda
    libffi-3.3                 |       hb1e8313_2          48 KB  anaconda
    libiconv-1.16              |       hca72f7f_2         1.3 MB  anaconda
    liblief-0.11.5             |       he9d5cce_1         2.0 MB  anaconda
    libxml2-2.9.12             |       hcdb78fc_0         1.2 MB  anaconda
    lz4-c-1.9.3                |       h23ab428_1         162 KB  anaconda
    markupsafe-2.0.1           |   py39h9ed2024_0          21 KB  anaconda
    nbformat-5.3.0             |   py39hecd8cb5_0         143 KB  anaconda
    ncurses-6.3                |       hca72f7f_2         1.1 MB  anaconda
    openssl-1.1.1n             |       hca72f7f_0         3.5 MB  anaconda
    pip-21.2.4                 |   py39hecd8cb5_0         2.0 MB  anaconda
    pkginfo-1.8.2              |     pyhd3eb1b0_0          27 KB  anaconda
    psutil-5.8.0               |   py39h9ed2024_1         349 KB  anaconda
    py-lief-0.11.5             |   py39he9d5cce_1         1.3 MB  anaconda
    pycosat-0.6.3              |   py39h9ed2024_0         114 KB  anaconda
    pycparser-2.21             |     pyhd3eb1b0_0          94 KB  anaconda
    pyopenssl-22.0.0           |     pyhd3eb1b0_0          49 KB  anaconda
    pyrsistent-0.18.0          |   py39hca72f7f_0          93 KB  anaconda
    pysocks-1.7.1              |   py39hecd8cb5_0          30 KB  anaconda
    python-3.9.12              |       hdfd78df_0        12.8 MB  anaconda           ⇦ 🤔
    python-dateutil-2.8.2      |     pyhd3eb1b0_0         241 KB  anaconda
    python-fastjsonschema-2.15.1|     pyhd3eb1b0_0          27 KB  anaconda
    python-libarchive-c-2.9    |     pyhd3eb1b0_1          50 KB  anaconda
    python.app-3               |   py39hca72f7f_0          16 KB  anaconda
    pytz-2021.3                |     pyhd3eb1b0_0         224 KB  anaconda
    pyyaml-6.0                 |   py39hca72f7f_1         172 KB  anaconda
    readline-8.1.2             |       hca72f7f_1         390 KB  anaconda
    requests-2.27.1            |     pyhd3eb1b0_0          52 KB  anaconda
    ripgrep-12.1.1             |                0         1.4 MB  anaconda
    ruamel_yaml-0.15.100       |   py39h9ed2024_0         266 KB  anaconda
    setuptools-61.2.0          |   py39hecd8cb5_0         1.3 MB  anaconda
    six-1.16.0                 |     pyhd3eb1b0_1          19 KB  anaconda
    soupsieve-2.3.1            |     pyhd3eb1b0_0          33 KB  anaconda
    sqlite-3.38.2              |       h707629a_0         1.8 MB  anaconda
    tk-8.6.11                  |       h7bc2e8c_0         3.3 MB  anaconda
    tqdm-4.64.0                |   py39hecd8cb5_0         123 KB  anaconda
    traitlets-5.1.1            |     pyhd3eb1b0_0          82 KB  anaconda
    tzdata-2022a               |       hda174b7_0         121 KB  anaconda
    urllib3-1.26.9             |   py39hecd8cb5_0         178 KB  anaconda
    wheel-0.37.1               |     pyhd3eb1b0_0          31 KB  anaconda
    xz-5.2.5                   |       h1de35cc_0         282 KB  anaconda
    yaml-0.2.5                 |       haf1e3a3_0          85 KB  anaconda
    zlib-1.2.12                |       h4dc903c_2         116 KB  anaconda
    zstd-1.5.2                 |       hcb37349_0         837 KB  anaconda
    ------------------------------------------------------------
                                           Total:        84.9 MB
```

Reply:
Checked with @benoit-cty and these dependencies come with conda-build (according to its [meta.yaml](https://github.com/conda/conda-build/blob/1ed8da062f123e1fbca604df26b064b267a3d094/conda.recipe/meta.yaml) file).
> Python 3.9 seems also to be its current Python revision (cf. [setup.py](https://github.com/conda/conda-build/blob/1ed8da062f123e1fbca604df26b064b267a3d094/setup.py)).